lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -142,6 +142,13 @@ def get_virtual_facts(self):
             virtual_facts['virtualization_role'] = 'guest'
             return virtual_facts
 
+        chassis_vendor = get_file_content('/sys/devices/virtual/dmi/id/chassis_vendor')
+
+        if chassis_vendor == 'Bochs':
+            self.facts['virtualization_type'] = 'kvm'
+            self.facts['virtualization_role'] = 'guest'
```

Comment:
Would like confirmation the droplets are KVM guests.

Reply:
@siwyd  @alikins Such timing -- I was about to submit a PR against module_utils/facts/virtual/linux.py to check {{product_name}} to detect a DO droplet when I came across this PR.

$ cat /sys/devices/virtual/dmi/id/product_name
Droplet